core/services/directrequestocr/models.go

```diff
@@ -42,9 +42,9 @@ const (
 	// It won't be included in OCR2 reporting rounds any more.
 	TIMED_OUT
 
-	// TRANSMITTED request is one that passed through ShouldTransmitAcceptedReport phase of OCR2 reporting.
+	// IN_TRANSMISSION request is one that passed through ShouldAcceptFinalizedReport phase of OCR2 reporting.
```

Comment:
IN_TRANSMISSION does not correctly reflect the request state.
It might not get transmitted if ShouldTransmitFinalizedReport fails.
What about "FINALIZED"?

Reply:
I'm worried that FINALIZED will not be meaningful to readers not familiar with OCR phases. But you're right, IN_TRANSMISSION might wrongly suggest that we're already transmitting from this particular node... ugh .. naming ;) FINALIZED seem to be slightly better.